lib/shared/config/exclude-cli-args.js

```diff
@@ -0,0 +1,34 @@
+'use struct';
+
+var yargs = require('yargs');
+var camelCase = require('camelcase');
+var copyProps = require('copy-props');
+var cliOptions = require('../cliOptions');
+
+function excludeCliArgs(config) {
```

Comment:
I still don't like this.  If we are doing the property copying correctly (only copy files that exist and only those that we expect), we shouldn't need this.